apps/web/src/AppRoutes.tsx

```diff
@@ -58,20 +58,26 @@ import {
   WorkflowsTestPage,
 } from './studio/components/workflows';
 import { WorkflowsStepEditorPageV2 } from './pages/templates/editor_v2/TemplateStepEditorV2';
+import { IS_CLERK_AUTH_ENABLED } from './config/index';
```

Comment:
Let's make this vendor agnostic.

```suggestion
import { IS_EE_AUTH_ENABLED } from './config/index';
```